### PR TITLE
String improvements [Finishes #83648154]

### LINKED ISSUE #167: String improvements
Suggestions for string improvements. Also see https://github.com/Yoast/License-Manager/issues/56.

As an aside, note that all links to code.google.com seem to get redirected to different URLs at developers.google.com, so they should be updated to the true destination URL.

> Settings saved.

The default notice string within WP core is "Settings saved." - full stop, not exclamation mark. With a string so similar, it should probably be consistent as that in core.

> Yoast Google Analytics:

Inconsistent plugin name reference.

> Google Analytics Premium

Inconsistent plugin name reference.

> If you want to track custom dimensions, to for instance track page views per author or post type, you should upgrade to the %1$spremium version of Google Analytics by Yoast%2$s.

Slightly awkward phrasing with that "for instance" in there. Consider: "If you want to track custom dimensions like page views per author or post type, you should upgrade to the %1$spremium version of Google Analytics by Yoast%2$s."

> Do not use this feature if you use FeedBurner, as FeedBurner can do this automatically, and better than this plugin can. Check &lt;a href="https://support.google.com/feedburner/answer/165769?hl=en&amp;ref_topic=13075" target="_blank">this help page&lt;/a> for info on how to enable this feature in FeedBurner.

Remove comma after "automatically".
The only variant bit in that URL is the language value, so that should stay in, but the rest can be removed in favour of a placeholder: "Do not use this feature if you use FeedBurner, as FeedBurner can do this automatically, and better than this plugin can. Check %sen%sthis help page%s for info on how to enable this feature in FeedBurner.". Add a context to explain what the placeholders / that en represents.

> There where no changes to save, please try again.

Typo - "where" > "were".

> The premium version of Google Analytics for WordPress with more features &amp;amp; support.

Reference to old name for the plugin.
Remove the encoded ampersand and use the word "and" instead.

> Yoast Google Analytics: Dashboard

Inconsistent plugin name reference. "Google Analytics by Yoast: Dashboard"

> Track your E-Commerce data and transactions with this E-Commerce extension for Google Analytics.

Inconsistent terminology. Swap "E-Commerce" to "eCommerce" to match name of the referenced plugin.
Inconsistent plugin name, "Google Analytics by Yoast".

> E-Commerce tracking

Inconsistent with plugin name, "eCommerce".

> Yoast Google Analytics: Extensions

Inconsistent plugin name reference. "Google Analytics by Yoast: Extensions"

> You have not installed any extensions for Yoast Google Analytics, so there are no licenses to activate.

Inconsistent plugin name reference.

> Yoast Google Analytics: Settings

Inconsistent plugin name reference. "Google Analytics by Yoast: Settings"

> General settings

A string of "General Settings" already exist, so making capitalisation consistent would reduce the number of strings to translate.

> Track outbound click & downloads

Remove the ampersand and use "and" instead.

> This adds &lt;code>&lt;a href="http://code.google.com/apis/analytics/docs/gaJS/gaJSApi_gat.html#_gat._anonymizeIp" target="_blank">_anonymizeIp&lt;/a>&lt;/code>, telling Google Analytics to anonymize the information sent by the tracker objects by removing the last octet of the IP address prior to its storage.

Move the markup out of the string and use a placeholder instead. Code, anchor, URL and linked content won't change with language. Add context to explain placeholder.

> First enable Universal tracking in your Google Analytics account. How to do that, please read &lt;a href="http://kb.yoast.com/article/125-universal-analytics#utm_medium=kb-link&utm_source=gawp-config&utm_campaign=wpgaplugin" target="_blank">this guide&lt;/a> to learn how to do that.

Move the URL out of the string and use a placeholder instead.
Encode the ampersands in the link.
Wording is a bit clumsy. Try: "First enable Universal tracking in your Google Analytics account. Please read &lt;a href="%s" target="_blank">this guide&lt;/a> to learn how to do that.

> You have to enable the Demographics in Google Analytics before you can see the tracking data. We have a doc in our &lt;a href="http://kb.yoast.com/article/154-enable-demographics-and-interests-report-in-google-analytics/#utm_medium=kb-link&utm_source=gawp-config&utm_campaign=wpgaplugin" target="_blank">knowlegde base&lt;/a> about this feature.

Move the URL out of the string and use a placeholder instead.
Encode the ampersands in the link.
Correct the typo of "knowlegde" -> "knowledge".
Suggest avoiding the colloquial "doc" and change it to "We have a knowledge base article about this feature.".

> This allows you to set the domain that's set by &lt;code>setDomainName&lt;/code> for tracking subdomains, if empty this will not be set.

Move the markup out of the string and use a placeholder instead.
Split into new sentence after "subdomains".
Add comma after "empty".

> This adds a &lt;code>&lt;a href="http://code.google.com/apis/analytics/docs/gaJSApiCampaignTracking.html#_gat.GA_Tracker_._setAllowAnchor" target="_blank">_setAllowAnchor&lt;/a>&lt;/code> call to your tracking code, and makes RSS link tagging use a # as well.

Move the markup out of the string and use a placeholder instead.
Add a `<code>` around the "#" character, but insert that via a placeholder as well (it's not going to change with a different language).

> This adds a &lt;code>&lt;a href="http://code.google.com/apis/analytics/docs/gaJS/gaJSApiDomainDirectory.html#_gat.GA_Tracker_._setAllowLinker" target="_blank">_setAllowLinker&lt;/a>&lt;/code> call to your tracking code,  allowing you to use &lt;code>_link&lt;/code> and related functions.

Move the markup out of the string and use placeholders instead.
Remove extra space before "allowing".

> This allows you to set the domain that's set by &lt;code>setDomainName&lt;/code> for tracking subdomains. If empty, this will not be set.

Move the markup out of the string and use a placeholder instead.
Should the code here have an underscore prefix?
Can it be linked to https://developers.google.com/analytics/devguides/collection/gajs/methods/gaJSApiDomainDirectory#_gat.GA_Tracker_._setDomainName for consistency with other parameters?

> Not for the average user: this allows you to add a line of code, to be added before the &lt;code>trackPageview&lt;/code> call.

Move the markup out of the string and use a placeholder instead.
Add underscore prefix to code.
Link to documentation (with placeholder for URL).

> https://yoast.com/wordpress/plugins/google-analytics/#utm_source=wordpress&utm_medium=plugin&utm_campaign=wpgaplugin&utm_content=v504

URL from the plugin headers. Not ideal for translators to have that version number at the end there if it's going to change for each version (and I'm not sure it's kept up to date anyway?), but there's no easy way to make it translator-friendly. Is it key that you track this data from this solitary link?

> This plugin makes it simple to add Google Analytics to your WordPress blog, adding lots of features, eg. error page, search result and automatic clickout and download tracking.

Who uses WP as a "blog" anymore? :-) Consider changing that word to "site".
"eg." should be "e.g.".

Is "clickout" a real word? Could "outgoing link" or something similar be used instead? Avoid "click" if possible, since keyboard users on the site won't be clicking anything.

> Clicks &amp;amp; downloads will be tracked as events, you can find these under Content &amp;raquo; Event Tracking in your Google Analytics reports.

Replace encoded ampersand with "and" instead.
Replace "&amp;raquo;" with a "&amp;#xBB;" hexadecimal character reference per http://www.w3.org/TR/html-polyglot/#named-entity-references (CR) and http://www.w3.org/TR/charmod/#C048 (Rec).

> Not for the average user: this allows you to add a line of code, to be added before the &lt;code>trackPageview&lt;/code> call.

Move the markup out of the string and use a placeholder instead.

> Add &lt;code>_setAllowLinker&lt;/code>

Move the markup out of the string and use a placeholder instead. Change the i18n function to one that allows a context to describe the placeholder value.

> Anonymize IP's

No apostrophe needed for plural initialisms.

> the debugging and firebug scripts are only loaded for admins.

Use a capital for "firebug".
Use the correct WP role name - "administrators", not "admins".
This is a string unnecssarily split from "Note" string for the purpose of adding markup and a colon around the word "Note". It's fine to have simple markup in strings when around what would otherwise be plain text. Colon's should be within the string too, since other languages may use different punctuation. "&lt;strong>Note:&lt;/strong> the debugging and Firebug scripts are only loaded for administrators."

> Custom dimensions

Tab label - should be "Custom Dimensions".

> Debug mode
> Debug setings

Used in tab, and then tab content heading - could these both changed to "Debug"? We already know we're on a settings page, so the latter is half redundant anyway.

### PR DESCRIPTION
**Pull request by andizer**
Will probably fixes #167

### CODE REVIEW

**Comment by jdevalk** [comment]
+1 from me.

**Comment by tacoverdo** [comment]
Looks good!

File: frontend/views/tracking-universal.php [hunk 1]
```diff
@@ -20,4 +20,4 @@
 	?>
 
 </script>
-<!-- / Yoast Google Analytics -->
+<!-- / Google Analytics by Yoast -->
```
Comment (GaryJones) [inline frontend/views/tracking-universal.php]: The red icon at the end of this line suggests there's no clear linespace at the end of the file.

Comment (omarreiss) [inline frontend/views/tracking-universal.php]: Fixed in 5c4f7a53645b6c12504c413264288805498ed0e0

**Pull request merged**